frontend/src/metabase/visualizations/components/FunnelNormal.jsx

```diff
@@ -29,7 +29,11 @@ export default class FunnelNormal extends Component {
     const rows = settings["funnel.rows"]
       ? settings["funnel.rows"]
           .filter(fr => fr.enabled)
-          .map(fr => series[fr.originalIndex].data.rows[0])
+          .map(
+            fr =>
+              series.find(singleSeries => singleSeries.card.name === fr.key)
```

Comment:
should we reuse `findSeriesByKey`?

`keyForSingleSeries` will use `single.card.name` under the hood, so I figure it should be equivalent.
https://github.com/metabase/metabase/blob/cd98a2c3586c846eee0dfaf95c5dda026c12174f/frontend/src/metabase/visualizations/lib/settings/series.js#L12